### ISSUE TITLE
change main navigation default colour.

### ISSUE THREAD

**Issue by phillipgardiner** [open-sdg/open-sdg#1058]
After team discussion its been decided to change the main navigation default colour to black. Countries will then be able to customise the colour should they wish via the Sass variables available.

The black colour to be used is: #0b0c0c
